Neos.Flow/Classes/Mvc/Controller/AbstractController.php

```diff
@@ -178,7 +180,7 @@ public function addFlashMessage($messageBody, $messageTitle = '', $severity = Er
                 $message = new Error\Message($messageBody, $messageCode, $messageArguments, $messageTitle);
                 break;
         }
-        $this->controllerContext->getFlashMessageContainer()->addMessage($message);
+        $this->flashMessageService->getFlashMessageContainerForRequest($this->request)->addMessage($message);
```

Comment:
this was dont to better reflect what state of the controller context is used and that the controller context becomes less relevant.

Reply:
Maybe i should revert that change here and create another pr for this? 

https://github.com/kitsunet/flow-development-collection/pull/4/commits/af2f9ab7f35be2d31e18cbecc7838365828f3178

Reply:
https://github.com/neos/flow-development-collection/pull/3292